appveyor.yml

```diff
@@ -6,19 +6,32 @@ cache:
 - '%USERPROFILE%\.nuget\packages -> ReactWindows\**\project.json'
 - ChakraCore\Build\VcBuild -> ChakraCore\lib\Common\*.h
 
+install:
```

Comment:
Added an INSTALL section to do the following:

1) npm install (for Jasmine)
2) download the winium test driver ZIP
3) unzip the winium test driver ZIP